src/Simulation/TargetDefinitions/Decompositions/ExpFrac.qs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.Quantum.Intrinsic {
     /// Register to apply the given rotation to.
     @EnableTestingViaName("Test.TargetDefinitions.ExpFrac")
     operation ExpFrac (paulis : Pauli[], numerator : Int, power : Int, qubits : Qubit[]) : Unit is Adj + Ctl {
-        let angle = (PI() * IntAsDouble(numerator)) / IntAsDouble(2 ^ power);
+        let angle = (PI() * IntAsDouble(numerator)) / (2.0 ^ IntAsDouble(power));
```

Comment:
It wasn't clear to me how this change was related to negative exponentiation. Luckily, Cesar had the same question when you fixed RFrac and you gave him an excellent explanation :) I wonder however, whether it's worth a comment here _in the code_?

Reply:
Seems like a good idea :)